WeScan/Edit/EditScanViewController.swift

```diff
@@ -163,7 +163,7 @@ final class EditScanViewController: UIViewController {
         
         let croppedImage = UIImage.from(ciImage: filteredImage)
         // Enhanced Image
-        let enhancedImage = filteredImage.applyingAdaptiveThreshold()?.withFixedOrientation()
+        let enhancedImage = filteredImage.applyingAdaptiveThreshold()
```

Comment:
@Boris-Em do you know for which scenario we've added the fixed orientation method? I feel like this is potentially breaking something.

Reply:
@AvdLee I felt the same way!
However I doubled check but could not be able to trace any bugs.
I thought that it probably done as a feature however originalImage and croppedImage left out by mistake.

Reply:
Looping in @julianschiavo as he merged in the [PR](https://github.com/WeTransfer/WeScan/pull/80) containing the changes. Do you remember what cases we were solving?

Reply:
I can't remember why we did this specifically, but we had to work around quite a bit of orientation issues at some point.